src/getters/index.ts

```diff
@@ -301,24 +304,44 @@ export async function getAllSponsorships({
     return sponsorships
 }
 
-export async function getSponsorshipById(
+export async function getParsedSponsorshipById(
```

Comment:
`getSponsorshipById` was only used in the context of `ParsedSponsorship` so I merged the 2 utilities together as `getParsedSponsorshipById`.